tomcat8@.service

```diff
@@ -1,7 +1,7 @@
 # Systemd unit file for tomcat instances.
-# 
+#
 # To create clones of this service:
-# 0. systemctl enable tomcat8@name.service
+# 0. systemctl enable tomcat8-nojsvc@name.service
```

Comment:
I think in the comment, `tomcat8-nojsvc@name.service` should be renamed to `tomcat8@name.service.